suite-common/logger/src/logsSlice.ts

```diff
@@ -0,0 +1,42 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export type LogEntry = { datetime: string; type: any; payload?: Record<any, any> };
+
+type LogsSliceState = {
+    logs: LogEntry[];
+};
+
+type LogsSliceRootState = {
+    logs: LogsSliceState;
+};
+
+const logsSliceInitialState: LogsSliceState = {
+    logs: [],
+};
+
+const MAX_ENTRIES = 200;
+
+export const logsSlice = createSlice({
+    name: 'logs',
+    initialState: logsSliceInitialState,
+    reducers: {
+        addLog: (
+            state,
+            action: PayloadAction<{ action: any; payload: Record<string, unknown> | undefined }>,
```

Comment:
AnyAction from @reduxjs/toolkit can be used or there should be a check that `action.payload.action` exists and that the action.payload.action.type is string
```suggestion
            action: PayloadAction<{ action: AnyAction; payload: Record<string, unknown> | undefined }>,
```

Reply:
https://github.com/trezor/trezor-suite/pull/7359/commits/3cc634793eec06a7461fbad15de709a91906b19b